api/src/test/scala/hmda/api/http/InstitutionSpec.scala

```diff
@@ -0,0 +1,40 @@
+package hmda.api.http
+
+import java.io.File
+
+import akka.http.scaladsl.model.{ ContentTypes, HttpEntity, Multipart }
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import com.typesafe.config.ConfigFactory
+import hmda.api.RequestHeaderUtils
+import hmda.persistence.HmdaSupervisor
+import hmda.persistence.HmdaSupervisor.FindActorByName
+import hmda.persistence.demo.DemoData
+import hmda.persistence.institutions.InstitutionPersistence
+import org.iq80.leveldb.util.FileUtils
+import org.scalatest.{ BeforeAndAfterAll, MustMatchers, WordSpec }
+
+trait InstitutionSpec extends WordSpec with MustMatchers with BeforeAndAfterAll
+    with ScalatestRouteTest with RequestHeaderUtils with InstitutionsHttpApi {
+
+  override def beforeAll(): Unit = {
+    val supervisor = HmdaSupervisor.createSupervisor(system)
+    supervisor ! FindActorByName(InstitutionPersistence.name)
+    DemoData.loadTestData(system)
+    super.beforeAll()
+  }
+
+  override def afterAll(): Unit = {
+    system.terminate()
+    val config = ConfigFactory.load()
+    val snapshotStore = new File(config.getString("akka.persistence.snapshot-store.local.dir"))
+    FileUtils.deleteRecursively(snapshotStore)
+  }
+
+  def multiPartFile(contents: String, fileName: String) =
```

Comment:
This function is used both in `UploadPathSpec` but also in `InstitutionHttpApiSpec`, that's why I put it in `InstitutionSpec`.